net-news/liferea/liferea-1.12.5.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GNOME2_EAUTORECONF="yes"
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit gnome2 pax-utils python-single-r1
```

Comment:
Hmm, I don't understand why it's using gnome2. After all, it looks more like a regular GTK+ package that has little to do with GNOME.

Reply:
Actually with gnome2 eclass custom ```src_configure``` and ```src_install``` ebuild functions should be removed.
Eclass is used in configure stage to pass ```disable-schemas-compile``` option and in install stage to prune static library.
Without eclass we should keep at least ```src_configure```.

Reply:
> Yes, you will need to do both manually.

... could you explain this please?
gnome2 eclass also used to update icons cache, glib schemas.

I strongly against dropping eclass and writting all this stuff manually without serious reason

Reply:
Yes, I'm aware that gnome2.eclass is GNOME team's old automated coffee maker with a lot of automagic random stuff that's completely irrelevant for your package.

Reply:
Sorry, but I feel this a bit unconstructive.
I beleave you don't like gnome2.eclass for some reason (as games.eclass some time ago), but I do not know all background and do not understand why I can not use eclass, that works for years.

If gnome2.eclass is deprecated or there some requirements exists which oblige ebuild to use all eclass 'automagic' stuff or nothing - please point me to that.
Otherwise feel free to reject this PR.

Anyway thanks for your time.

Reply:
One of the reasons for not using big blobs of automagic is that you don't even notice that you've broken them. In this particular case, by not calling `gnome2_pkg_postinst`.

Reply:
Thanks for pointing to ```gnome2_pkg_postinst```. I added ```networkmanager``` USE flag, but forget to clean ```pkg_postinst```.
I was wrong saying that this ebuild used "some" gnome2.eclass features. I carefuly reviewed gnome2.eclass with projection to liferea and the truth is that no gnome2.eclass features are needed for liferea build process at all I beleave.
So minimal "clean" ebuild is fine without gnome2.eclass.

Reply:
Sadly, ```glib-compile-schemas``` is called by liferea in sandboxed ```src_install``` so it should be used in unsandboxed stages.
Because of that I prefer to use "automagic" eclass for that stuff instead of writting it by hand. I searched tree for any eclass not using gnome2.eclass for that and found none.
So I reverted gnome2.eclass cleanup.